switch_model/transmission/transport/build.py

```diff
@@ -193,18 +193,25 @@ def define_components(mod):
         'trans_length_km', 'trans_efficiency', 'existing_trans_cap')
     mod.trans_new_build_allowed = Param(
         mod.TRANSMISSION_LINES, within=Boolean, default=True)
+    mod.trans_capital_cost_per_mw_km = Param(
+        within=NonNegativeReals,
+        default=1000)
```

Comment:
Is this a real value or just a test value? Also, we are always providing this number, right? Therefore the default will be used in particular cases where we did not specify the cost?

Reply:
See response below